components/dropdown/dropdown-content-styles.js

```diff
@@ -108,6 +108,13 @@ export const dropdownContentStyles = css`
 		width: 100vw;
 	}
 
+	d2l-focus-trap {
+		max-width: 370px;
+		min-width: 70px;
+		position: absolute;
+		width: 100vw;
+	}
```

Comment:
Interesting -- why was this needed?

Reply:
Ah, some of those styles were left over from previous debugging, I can remove them. The position: absolute is needed, though. In the non-focus case, .d2l-dropdown-content-width has the same position. It also needs to be applied to the focus-trap, or else the positioning is wrong and the pointer does not line up with the container.